src/node/hooks/express/static.js

```diff
@@ -66,6 +66,7 @@ exports.expressCreateServer = async (hookName, args) => {
       delete clientPlugins[name].package;
     }
     res.header('Content-Type', 'application/json; charset=utf-8');
```

Comment:
Looks like `.header()` is an old Express 3.x method. Might as well update to the new hotness while you're here:

```suggestion
    res.setHeader('Content-Type', 'application/json; charset=utf-8');
```